dcgpy/_graphviz_visualize.py

```diff
@@ -0,0 +1,132 @@
+def _graphviz_visualize(self, erc = 0, draw_inactive = True):
+    """
+    ex.visualize(self, erc, draw_inactive)
+
+    Visualizes the d-CGP expression
+
+    Visualizes the graph of the d-CGP expression, by generating a png image and displaying it on Matplotlib axes
+
+    Note:
+        This method requires matplotlib and pygraphviz modules installed in your Python system
+
+    Args:
+        erc (an ``int``): number of ephemeral random consants in input
```

Comment:
here explain what the erc does, i.e. considers some of the input as symbols c_n rather than x_n (i guess)